lms/djangoapps/support/static/support/jsx/entitlements/data/reducers/entitlements.js

```diff
@@ -0,0 +1,17 @@
+import { entitlementActions } from '../actions/actionCreators';
+
+function entitlements(state = [], action){
+  switch(action.type){
+    case entitlementActions.fetch.SUCCESS:
+      //TO BE REMOVED when entitlements in store are displayed in table
+      console.log('Fetching entitlements success ', action.entitlements)
+      return action.entitlements
+    case entitlementActions.fetch.FAILURE:
+      console.log('Fetching entitlements failed ', action.error)
+      return state
```

Comment:
If this `case` is returning `state` does it need to be explicitly handled?

Reply:
Right now it does because it feels like the correct place to log the error, in the future I can do something more smart like change an 'error_message' value in the state or something.